app/assets/stylesheets/shared/components/_site-nav.scss

```diff
@@ -125,6 +125,11 @@
   }
 }
 
+.link--border{
```

Comment:
Nothing blocking, but I think this would be `.link-border` or `.site-link__border` and a space between the class and the curly bracket. :)

Reply:
Resolved in 1100b01